extensions/widgets/top-bar/src/components/topbar.tsx

```diff
@@ -110,6 +111,8 @@ const Topbar: React.FC<ITopbarProps> = props => {
   const customStyle =
     'flex-row justify-between items-center py-1.5 px-2 space-x-4 xs:(fixed top-0 z-8)';
 
+  const isAlpha = true;
```

Comment:
This one is quite confusing. Why not just have the Pill component for now and when we need to remove it we will replace it with the Text component ? That way things will be a lot cleaner.